bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapJSon.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.shelly.internal.coap;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.lang.Validate;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * The {@link ShellyCoapJSon} helps the CoIoT Json into Java objects
+ *
+ * @author Markus Michels - Initial contribution
+ */
+public class ShellyCoapJSon {
+
+    public static final String CoIoT_Tag_blk = "blk";
```

Comment:
Use Java naming conventions:
```suggestion
    public static final String COIOT_TAG_BLK = "blk";
```

Reply:
done